.github/CONTRIBUTING.md

```diff
@@ -18,24 +18,23 @@ you alongside!
 
 #### Table of Contents
 
-- [Code of Conduct](#code-of-conduct)
-- [What you need to know before getting started](#what-you-need-to-know-before-getting-started)
-  - [Interface X and packages](#interface-x-and-packages)
-- [How to contribute](#how-to-contribute)
-
-  - [Report bugs](#report-bugs)
-  - [Suggest enhancements](#suggest-new-features-or-enhancements)
-  - [Contribute code for the first time](#contribute-code-for-the-first-time)
-  - [Set up your development environment](#set-up-your-development-environment)
-  - [Pull requests](#pull-requests)
-
-- [Style guides](#style-guides)
-  - [Git commit messages](#git-commit-messages)
-  - [JavaScript style guide](#javascript-style-guide)
-  - [Documentation style guide](#documentation-style-guide)
-  - [Testing style guide](#testing-style-guide)
-- [Any questions?](#any-questions)
-- [License](#license)
+- [Contributing Guidelines](#contributing-guidelines) - [Table of Contents](#table-of-contents)
+  - [Code of Conduct](#code-of-conduct)
+  - [What you need to know before getting started](#what-you-need-to-know-before-getting-started)
+    - [Interface X and Packages](#interface-x-and-packages)
+  - [How to contribute](#how-to-contribute)
+    - [Report Bugs](#report-bugs)
+    - [Suggest new features or enhancements](#suggest-new-features-or-enhancements)
+    - [Contribute code for the first time](#contribute-code-for-the-first-time)
+    - [Set up your development environment](#set-up-your-development-environment)
+    - [Pull requests](#pull-requests)
+  - [Style guides](#style-guides)
+    - [Git commit messages](#git-commit-messages)
+    - [JavaScript style guide](#javascript-style-guide)
+    - [Testing style guide](#testing-style-guide)
+    - [Documentation style guide](#documentation-style-guide)
+  - [Any questions?](#any-questions)
+  - [License](#license)
```

Comment:
I don't see a reason to change those lines 🤔 I think in this file we only need to change the line 71. Please, undo this change.

Reply:
That's right. I undid this change.

Reply:
@yokishava I'm sorry I think I mislead you with my comment. I just wanted to undo the changes to those lines. We still need to change [this line](https://github.com/yokishava/x/blob/feature/rename-search-types/.github/CONTRIBUTING.md?plain=1#L70). `**[@empathyco/x-types](/packages/search-types)**` should be `**[@empathyco/x-types](/packages/x-types)**` 

Apart from that LGTM

Reply:
Thak you for your review. I'm sorry. I also restored the x-types path by mistake. Corrected again.